apps/admin_api/priv/spec.yaml

```diff
@@ -1412,17 +1412,17 @@ paths:
           $ref: "#/components/responses/AccessKeyResponse"
         '500':
           $ref: "#/components/responses/InternalServerError"
-  /access_key.update:
+  /access_key.enable_or_disable:
```

Comment:
Should we use `deprecated: true` on the specs instead of removing?

It will show like this in the docs:

![image](https://user-images.githubusercontent.com/921194/48753158-6871d780-ecbf-11e8-8933-2b087b3b77ad.png)

Reply:
Oooh, that looks great! Didn't know we could do that.